supervision/keypoint/skeletons.py

```diff
@@ -1,11 +1,11 @@
 from enum import Enum
-from typing import Dict, List, Tuple
+from typing import ClassVar, Dict, List, Tuple
 
 Edges = List[Tuple[int, int]]
 
 
 class Skeleton(Enum):
-    COCO = [
+    COCO: ClassVar[List[Tuple[int, int]]] = [
```

Comment:
1. We can use `Edges` here.
2. Is `ClassVar` required here? Does it capture something about the contained data that would otherwise confuse MyPy or the user? In C++ terms - it feels more related to the `lvalue` than the `rvalue`. More about how the data is stored than what the data is.

Reply:
Reading up about mutable class var annotations

Reply:
This is an enum, so it would make sense for the values to be immutable. Can we try changing Edges to `Tuple[Tuple[int, int]]`? If there's issues somewhere, let's then keep `ClassVar[Edges]`.

Reply:
@LinasKo  done